src/wazuh_db/schema_upgrade_v10.sql

```diff
@@ -11,4 +11,33 @@
 UPDATE sca_check SET result='not applicable' WHERE status='Not applicable';
 ALTER TABLE sca_check DROP COLUMN status;
 
+CREATE TABLE IF NOT EXISTS _sys_programs (
```

Comment:
Move this to a new v11, because 4.5 is closed.

Reply:
I guess the failing checks are due to the fact that we still have 4.5 in master.